File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int size = nums.size();
        unordered_map<int , int> m;

        for(int i = 0 ; i < size ; i++){
            int n1 = nums[i];
            int n2 = target - nums[i];

            if(m.find(n2) != m.end()){
                return {i , m[n2]};
            }
            m[n1] = i;
    }
 
    return {0 , 0};
    }
};

File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int size = nums.Length;

        int[] ans = {-1 , -1};
        for(int i = 0 ; i<size ; i++){
            for(int j = i+1 ; j<size ; j++){
                int sum = nums[i]+nums[j];
                if(sum == target){
                    ans[0] = i;
                    ans[1] = j;
                    return ans;
                    
                } 
            }

        }

        return ans;
    }
}